src/build-data/makefile/header.in

```diff
@@ -19,7 +19,7 @@ CLI_FLAGS      = $(CXXFLAGS) $(WARN_FLAGS)
 TEST_FLAGS     = $(CXXFLAGS) $(WARN_FLAGS)
 
 SCRIPTS_DIR    = %{scripts_dir}
-INSTALLED_LIB_DIR = %{destdir}/%{libdir}
+INSTALLED_LIB_DIR = $(DESTDIR)/%{libdir}
```

Comment:
I think this change is wrong since INSTALLED_LIB_DIR is used before the `make install` step and only `make install` depends on DESTDIR. Is it `INSTALLED_LIB_DIR = %{prefix}/%{libdir}`?